docs/src/main/asciidoc/_developer-guide/EJB_invocations_from_a_remote_client_using_JNDI.adoc

```diff
@@ -332,7 +332,7 @@ name are as follows:
 *app-name* : This is the name of the .ear (without the .ear suffix) that
 you have deployed on the server and contains your EJBs.
 
-* Java EE 6 allows you to override the application name, to a name of
+* Jakarta EE 8 allows you to override the application name, to a name of
```

Comment:
WDYT about just dropping the '8' here and in most of the similar places where you're doing this kind of version correction?

If EE ever evolves such that the doc chunks with these #s included are no longer spec-required etc, there would need to be a doc change regardless.  I doubt having #s in the text in some spots would help much in finding places to correct, so I don't think the #s add much value.

Of course there are places where the # is required, e.g. anything where we described what spec version we support.

Reply:
makes sense, thanks.

Reply:
just updated locally, will push changes soon, commits should be squashed but will leave separate for now.